git/git.go

```diff
@@ -321,16 +327,23 @@ func (provider *repositoryProvider) snapshot(repository *git.Repository, commit
 	treeWalker := object.NewTreeWalker(tree, true, nil)
 	defer treeWalker.Close()
 
-	var indexOutputFile *os.File = nil
+	var indexOutputFile *csv.Writer = nil
 	if optionalIndexFilePath != "" && !dryRun {
 		locIndexOutputFile, err := os.Create(optionalIndexFilePath)
 		if err != nil {
 			return 0, fmt.Errorf("failed to create index file '%v': %v", optionalIndexFilePath, err)
 		}
 
+		csvWriter := csv.NewWriter(locIndexOutputFile)
+		csvWriter.Comma = '\t'
+		err = csvWriter.Write([]string{"Path", "BlobId", "IsFile"})
+		if err != nil {
+			return 0, fmt.Errorf("failed to write file headers '%v': %v", optionalIndexFilePath, err)
+		}
```

Comment:
did we write headers previously also?

Reply:
no, but this helps in C# to deserialize the data